nix/checks/core.nix

```diff
@@ -87,7 +90,9 @@ in
 
         virtualisation.vlans = [ 1 ];
         systemd.services.systemd-networkd.environment.SYSTEMD_LOG_LEVEL = "debug";
+        networking.useDHCP = false;
         systemd.network = {
+          enable = true;
```

Comment:
We need to enable systemd.network if not consuming from `nixos-configurations/core-master/configuration.nix`